lrauv_ignition_plugins/worlds/empty_environment.sdf

```diff
@@ -238,7 +238,7 @@
           <property type="double" key="x">400</property>
         </ignition-gui>
       </plugin>
-      <plugin filename="VisualizePointCloud" name="Visualize science data">
+      <plugin filename="PointCloud" name="Visualize science data">
```

Comment:
Should we just comment this plugin out in this world file and add a comment, since the science data doesn't work with spawning vehicles yet (see comment in lines 57-58)? I had forgotten about the state of that and tried to test the plugin in this file, which of course doesn't work as expected.

Reply:
Oh I thought that was working, let me take a look

Reply:
So, it was working, but the QOL improvements from `buoyant_tethys` weren't on `empty_environment` yet. Fixed in 954163e. I also changed the science data plugin so it only advertises topics once the science data is available. Let me know if it works for you now.

Reply:
Ah, now it works. Thanks for the fixes and enhancement! (That server config file... ha)